evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -674,12 +674,16 @@ contract FluxAggregator is AggregatorInterface, Owned {
   function roundState(address _oracle)
     external
     view
-    returns (uint32 _reportableRoundId, bool _eligibleToSubmit, int256 _latestRoundAnswer)
+    returns (uint32 _reportableRoundId, bool _eligibleToSubmit, int256 _latestRoundAnswer, uint64 _timesOutAt)
   {
-    uint32 roundId = rounds[reportingRoundId].details.answers.length == rounds[reportingRoundId].details.maxAnswers || timedOut(reportingRoundId)
-      ? reportingRoundId.add(1)
-      : reportingRoundId;
-    return (roundId, eligibleToSubmit(_oracle), rounds[latestRoundId].answer);
+    bool finishedOrTimedOut = rounds[reportingRoundId].details.answers.length == rounds[reportingRoundId].details.maxAnswers || timedOut(reportingRoundId);
+    uint32 reportableRoundId = finishedOrTimedOut ? reportingRoundId.add(1) : reportingRoundId;
+    return (
+      reportableRoundId,
+      eligibleToSubmit(_oracle),
+      rounds[latestRoundId].answer,
+      finishedOrTimedOut ? 0 : rounds[reportableRoundId].startedAt + rounds[reportableRoundId].details.timeout
```

Comment:
Its hard to know whether timeout is a duration of a time like 'startedAt', can this be fixed?  is there a standard way to denote durations?

Reply:
Maybe worth chatting about separately -- it would be out of scope for this PR, since timeout wasn't introduced here.

Reply:
Actually, thats what I was trying to suggest ;)